src/app/utils/expect-entity-data.spec.ts

```diff
@@ -45,6 +45,7 @@ export function expectEntitiesToMatch(
   // write arrays to console for easier debugging of complex mismatching objects
   console.log("expected objects", cleanExpected);
   console.log("actual objects", cleanActual);
+  isSame("entities", cleanExpected, cleanActual);
```

Comment:
The main reason for this check is to have the difference between two objects be more clear. What this should do is print out where and how an object is different, instead of printing out the expected and actual objects which might be very big and unobvious. So say you have two objects `{name: "A", id: "B", addres: { postocde: 7293, ... }}` and `{name: "A", id: "B", addres: { postocde: 4812, ... }}` this method should say that the objects differ only at the postcode